Make fails trying to remove tools/tek4010/tek4010

```
MAIN BUILD SCRIPT DONE
Tue 24 Nov 2020 06:03:49 AM EST
touch out/pdp10-kl/stamp
rm tools/tek4010/tek4010
rm: cannot remove 'tools/tek4010/tek4010': No such file or directory
make: *** [Makefile:290: tek-hack] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -287,7 +287,7 @@ $(VT52):
 	$(MAKE) -C tools/vt05 vt52
 
 tek-hack:
-	rm $(TEK)
+	rm -f $(TEK)
 
 $(TEK): tek-hack
 	$(MAKE) -C tools/tek4010 tek4010

